src/sentry/conf/server.py

```diff
@@ -664,6 +665,7 @@ def create_partitioned_queues(name):
     'sentry.interfaces.Query': 'sentry.interfaces.query.Query',
     'sentry.interfaces.Http': 'sentry.interfaces.http.Http',
     'sentry.interfaces.User': 'sentry.interfaces.user.User',
+    'sentry.interfaces.CSP': 'sentry.interfaces.csp.CSP',
```

Comment:
Nevermind, just did it. It fits the naming pattern of all the others, specifically like `Http`.